src/util.ts

```diff
@@ -49,9 +49,26 @@ export function ucfirst(str: string) {
     return firstLetter.toUpperCase() + str.substr(1);
 }
 
-export function debug(msg: any) {
-    if (isDev) {
-        console.log(msg);
+// log levels = info / warn / error / debug
+export function log(module: string, msg: any, level:string = "info") {
```

Comment:
Please add Log.Info(), Log.Debug(), Log.Warn(), Log.Error()

Reply:
Not sure what you mean. The introduced log function has all those log levels. you provide them through the `level` param.

Or do you mean separate functions `log_info(module, msg)`, `log_debug(module, msg)`, `log_warn(module, msg)` and `log_warn(module, msg)`? I'd vote against that (not that I have a say of course) as that would require to import all of those functions instead of only importing the `log` function.

Reply:
Separate functions

Because someone could just forget about adding optional `level` parameter to `log()`

Reply:
> Separate functions
> 
> Because someone could just forget about adding optional `level` parameter to `log()`

Still think this is not the best choice but then again - time is precious. I'll dig in and make those changes.